xbmc/GUIInfoManager.cpp

```diff
@@ -9089,20 +9089,21 @@ void CGUIInfoManager::ResetCurrentItem()
 void CGUIInfoManager::UpdateInfo(const CFileItem & item)
 {
   m_currentFile->UpdateInfo(item);
+  if (item.IsVideo())
+    m_currentMovieThumb = item.GetArt("thumb");
 }
 
 void CGUIInfoManager::SetCurrentItem(const CFileItem &item)
 {
   ResetCurrentItem();
 
-  CFileItem newItem(item);
+  *m_currentFile = item;
```

Comment:
@ksooo thanks, I pulled this up to this level

Reply:
SetCurrentSong and SetCurrentGame some lines below also set m_currentFile, which could be removed now.